{"uuid": "e66c7778-9af6-4964-8912-faec61bd83d7", "children": ["045bb27e-2547-46ae-b2d5-ff5e1fa82531", "d5f65e01-22d8-4a93-8c8e-667fb7980b53", "2d1dbc75-a14a-43cc-8efc-ff2469ed0b36", "b9ed716c-759f-4a59-9811-b481ba247f93", "a3d83b25-27ab-42e1-b7f2-0d4365161c74"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: create container: Error response from daemon: No such image: selenoid/vnc_chrome:96.0\n\n", "trace": "  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/bender/work/ui_api_autotest/conftest.py\", line 51, in browser\n    browser = webdriver.Remote(\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 268, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 359, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 424, in execute\n    self.error_handler.check_response(response)\n  File \"/home/bender/work/ui_api_autotest/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1643056195066, "stop": 1643056210101}], "start": 1643056195066, "stop": 1643056213189}